package.json

```diff
@@ -3,8 +3,8 @@
   "description": "Snyk CLI NuGet plugin",
   "main": "dist/index.js",
   "scripts": {
-    "test": "npm run lint && npm run test:unit",
-    "test:unit": "jest --coverage --silent=false --detectOpenHandles",
```

Comment:
`--detectOpenHandles` is a sign that something's wrong and forces tests to be run in parallel. It's possible that there has been problems in the past, maybe fixed in [my latest refactor](https://github.com/snyk/snyk-nuget-plugin/pull/149), but I see no issues now.

Reply:
Seems to have cut test time at least by half: https://app.circleci.com/pipelines/github/snyk/snyk-nuget-plugin/236/workflows/e4d23234-3cbe-4c9e-aa5e-0baf69598911